en/editor/animation/animation.md

```diff
@@ -12,7 +12,7 @@ To use `Animation`, please refer to the [Animation API](__APIDOC__/en/classes/an
 
 ## Clip Animation clip
 
-An __Animation Clip__ is a piece of declaration data for an animation. If we mount it on an __Animation Component__, we can apply this animation data to a node. An __Animation Component__ can mount multiple __Animation Clips__. It is not possible to edit animations on nodes without clips. After clicking on the relevant nodes, if the node has only __Animation Components__, but no clip files, the __Animation Editor__ will display the **Create Clip File** button, __click__ to create and assign to the component. In the animation editing mode, you can also switch to edit different __Animation Clips__ through the clip option in the lower left corner of the __Animation Editor__.
+An __Animation Clip__ is a piece of declaration data for an animation. If we mount it on an __Animation Component__, we can apply this animation data to a node. An __Animation Component__ can mount multiple __Animation Clips__. It is not possible to edit animations on nodes without clips. After clicking on the relevant nodes, if the node has only __Animation Components__, but no clip files, the __Animation Editor__ will display the **Create Clip File** button, __click__ to create and assign to the component. In the animation editing mode, you can also switch to edit different __Animation Clips__ through the clip option in the lower left corner of the __Animation Editor__. And the attribute modification that is not recorded in the animation key frame will be restored after exiting __animation editing mode__.
```

Comment:
If we mount it on an
-->
If attached on an

Reply:
mount
-->
attach

replace any instances in this document

Reply:
And the attribute modification that is not recorded in the animation key frame will be restored after exiting __animation editing mode__.
-->
The attribute modifications that are not recorded in the animation key frame will be restored after exiting __animation editing mode__.